voyager/plugins/event-source/ethereum/src/main.rs

```diff
@@ -201,14 +198,15 @@ impl Module {
             .await
             .map_err(json_rpc_error_to_error_object)?;
 
+        let other_port_id = self_channel.counterparty.port_id.clone();
```

Comment:
nit `counterparty_*` for both

Reply:
i specifically use other_ as opposed to counterparty_ to avoid confusion in places where there is no "counterparty" (i.e. packets with src/dst)